### ISSUE TITLE
'Enrol Confirm' not reachable from menu?

### ISSUE THREAD

**Issue by gtoffoli** [success4all-eu/learning-platform#7]
In the teacher role for the test course "Resources aggregration demo", I've received via SUCCESS-Moodle a notification email with the following text:

> Disciplina: Resources aggregration demo
> Nome/Apelido: Rui Teles
> Observações:
> [Pedidos de inscrição](https://learn.success4all.eu/enrol/apply/manage.php?id=166)

Since clicking on the link in my PC opens it with FF, while I prefer accessing SUCCESS-Moodle with Chrome, I logged-in with Chrome and:
- didn't see any prompt for action
- wasn't able to find, by navigating with various menus, a page where pending actions (in general, or related to the specific course) are listed; for example I reached the page _My Courses > Dashboard > RAD > Participants_, which provides a lot of options but not what I was looking for.

On the other hand, copying to the address box of Chrome the link from the notification email, I correctly was taken to the _Enrol Confirm_ page for the course .

In conclusion, I found that possibly the menus are overcrowded and they allow to get to the same page/function following multiple paths, but possibly some useful paths are missing. Also, at log-in, prompts for actions or a link to a list of pending tasks would be very useful.

**Comment by michzimny** [comment]
@gtoffoli, I think it's a very important suggestion, thank you!

We managed to enable notifications also within the Moodle page:
![image](https://user-images.githubusercontent.com/1483196/184366349-1411feed-c2f2-41dd-9f6f-252dabf29af7.png)

It says "New enrolment request". One should see it in their language. In the details, there's a link to the management view.

Hope it helps!
